metrics/inspector_prioritization.py

```diff
@@ -0,0 +1,274 @@
+import boto3
+import pandas as pd
+import numpy as np
+from arcgis.gis import GIS
```

Comment:
i had to install `arcgis` in order to get this running—looks like it's missing from the requirements.

annoyingly, i could only install it after pulling the amd64 version of the image. so this:
```shell
docker run -it --env-file env_file --platform linux/amd64 -v "$(pwd):/app" atddocker/dts-right-of-way-reporting:production /bin/bash
```

then this

```shell
pip install arcgis
```

Reply:
@johnclary I was able to find a patch for the dockerfile to get it to build for me